apps/dsb-message-broker/src/auth/auth.service.ts

```diff
@@ -16,31 +16,33 @@ export class AuthService implements OnModuleInit {
         const mbDID = this.configService.get<string>('MB_DID');
         const rpcUrl = this.configService.get<string>('WEB3_URL');
         const cacheServerUrl = this.configService.get<string>('CACHE_SERVER_URL');
+        const { signerService, connectToCacheServer } = await initWithPrivateKeySigner(
```

Comment:
This should help you set the `cacheServerUrl`.
You probably need to import `setCacheClientOptions`. There are some docs here (though they don't have much more than what I added) https://github.com/energywebfoundation/iam-client-lib#initialization
```suggestion
       const chainId = this.configService.get<string>('CHAIN_ID'); // should be 73799 for Volta
        setCacheClientOptions(chainId, {
            url: 'https://identitycache-staging.energyweb.org/v1',
            cacheServerSupportsAuth: true,
        });
        const { signerService, connectToCacheServer } = await initWithPrivateKeySigner(
```

Reply:
ah, yep, thanks, Chris S was telling me how as well. i ll try that.